strategy/tests/test_actions.py

```diff
@@ -26,6 +26,74 @@ def test_pivot_with_ball():
     assert(goal_pos[2] > start_pos[2])
     assert(np.linalg.norm(goal_pos[:2] - ball_pos) < gs.ROBOT_RADIUS * CLOSE_ENOUGH_FACTOR)
 
+def test_charge_up_to():
+    """ Tests charge_up_to in the trivial case of zero kick speed.
+    Passes if charging finishes immediately.
+    """
+    simulator = Simulator("clear_field_test")
+    simulator.pre_run()
+    gs = simulator.gs
+    strategy = Strategy(team, strategy_name)
+    strategy.gs = gs
+    charge = strategy.charge_up_to(1, 0.0)
+    assert(charge)
+
+def test_kick_ball():
+    """Tests kick_ball to make sure it sets is_kicking to true"""
+    simulator = Simulator("clear_field_test")
+    simulator.pre_run()
+    gs = simulator.gs
+    strategy = Strategy(team, strategy_name)
+    strategy.gs = gs
+    strategy.kick_ball(1)
+    commands = gs.get_robot_commands(team, 1)
+    assert(commands.is_kicking)
+
+def test_set_dribbler():
+    """ Tests set_dribbler after setting to True and to False.
+    Passes if it correctly returns the value that it is set to.
+    """
+    simulator = Simulator("clear_field_test")
+    simulator.pre_run()
+    gs = simulator.gs
+    strategy = Strategy(team, strategy_name)
+    strategy.gs = gs
+    strategy.set_dribbler(1, True)
+    commands = gs.get_robot_commands(team, 1)
+    assert(commands.is_dribbling)
+    strategy.set_dribbler(1, False)
+    commands = gs.get_robot_commands(team, 1)
+    assert(not commands.is_dribbling)
+
+def test_stop():
+    """ Tests stop function.
+    Passes if robot has no waypoints and is set to be remain stationary.
+    """
+    simulator = Simulator("clear_field_test")
+    simulator.pre_run()
+    gs = simulator.gs
+    strategy = Strategy(team, strategy_name)
+    strategy.gs = gs
+    strategy.path_find(1, [0, 0, 0])
+    strategy.stop(1)
+    commands = gs.get_robot_commands(team, 1)
+    assert(not commands.waypoints)
```

Comment:
no parens needed for assert in python (It may cause errors if you use an assert message after